cfme/tests/openstack/infrastructure/test_hosts.py

```diff
@@ -0,0 +1,112 @@
+import pytest
+from utils import testgen
```

Comment:
Leave an empty line between the types of imports (builtin, installed, local). Move utils.testgen down with the other utils.wait import so things are more organized. If you're feeling extra pedantic organize all the local imports into alphabetical.